src/app/app.component.html

```diff
@@ -3,11 +3,11 @@
   <section>
     <ng-progress [color]="'red'" [spinner]="false"></ng-progress>
 
-    <app-header *ngIf="currentUrl" [layoutState]="layoutState$ | async"></app-header>
+    <app-header *ngIf="initialized" [layoutState]="layoutState$ | async"></app-header>
 
     <main class="main-body">
       <router-outlet></router-outlet>
     </main>
-    <app-footer *ngIf="currentUrl"></app-footer>
+    <app-footer *ngIf="initialized"></app-footer>
   </section>
```

Comment:
Actually there is no need for `*ngIf="initialized"` at all. Since the code was an year old related to `currentUrl`, that was used to set diff headers as per URL. But we missed to remove it later after UI was completely changed.
So no need for any conditional view here. You can remove `initialized` variable as well.

In short this Pr will only remove previously dead code.

Thanks @Zackazt